starter-code/About.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <title>Joe Jphnson | Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <header>
+        <hr class="bordLine">
+        <div class="txtCnter">
+            <ul>
+                <li class="list-item"> <a href="Portfolio.html">AboutMe</a></li>
+                <li class="list-item"> <a href="About.html">Portfolio</a></li>
+            </ul>
```

Comment:
This got me confused haha. So Portfolio.html is redirecting to the About me page and the About.html redirects to the portfolio page. This is fine for now but always make sure that the links are redirecting to the correct path :+1: